src/components/Greetings.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import axios from 'axios';
+import { getMessage } from '../redux/greetingSlice';
+
+const Greetings = () => {
+  const dispatch = useDispatch();
+  const message = useSelector((state) => state.greeting);
+  async function fetchMessage() {
+    await axios
+      .get('http://127.0.0.1:3000/api/v1/messages')
+      .then((response) => {
+        dispatch(getMessage(response.data.greeting));
+      });
+  }
+  useEffect(() => {
+    fetchMessage();
```

Comment:
* Kindly fix this error I get when I try to display random greetings below is a screenshot, I also suggest you use your action reducer to make your API calls and update the store properly 
![image](https://user-images.githubusercontent.com/52497246/199649752-2502237d-ca93-478b-b821-07c4f7d3b912.png)

Reply:
hello @Davidosky007 it is fetching from port 3000 so the backend should be running on port 3000 it is working on my side
![Screenshot from 2022-11-03 08-02-21](https://user-images.githubusercontent.com/86611927/199651043-573a4670-b7be-4209-85a1-15575c9088d5.png)

Reply:
well resolved! ✔️
![image](https://user-images.githubusercontent.com/12745474/199665343-7d913f59-db36-48be-9837-01704fa086d7.png)